plugins/modules/net_tools/nmcli.py

```diff
@@ -69,6 +69,7 @@
         description:
             - The IPv4 address to this interface.
             - Use the format C(192.0.2.24/24).
+            - If defined, automatically set C(ipv4.method) to C(manual). If the I(method4) parameter is used, this parameter is ignored.
```

Comment:
```suggestion
            - If defined and I(method4) is not specified, automatically set C(ipv4.method) to C(manual).
```

Reply:
Perfect ! I will merge this and do the same for ip6 😊